.gitignore

```diff
@@ -18,10 +18,10 @@
 *.MOV
 *.avi
 *.data
-*.json
 *.cfg
 !setup.cfg
 !cfg/yolov3*.cfg
+database.json
```

Comment:
shouldn't this be database/database.json?

Reply:
Like this I exclude every database, also in case they move it around.
It seemed safer, but let me know what you think

Reply:
oh I thought this only works for database.json files in the root